packages/flutter/lib/src/material/dropdown.dart

```diff
@@ -429,8 +451,9 @@ class _DropdownRoutePage<T> extends StatelessWidget {
       padding: padding.resolve(textDirection),
     );
 
-    if (theme != null)
+    if (theme != null) {
```

Comment:
Previous syntax was fine: https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo#separate-the-if-expression-from-its-statement